### PR TITLE
Add: Module actions

### PR DESCRIPTION
**Pull request by Gorgeous-Patrick**
## Describe your changes
1. Set wait time between tasks to be 0
2. Add module actions to locust test
3. Configured jaseci.yaml so that the pod is suitable for module actions.

### CODE REVIEW

**Comment by Gorgeous-Patrick** [comment]
@ypkang Ready for review.

File: scripts/jaseci.yaml [hunk 1]
```diff
@@ -17,11 +17,12 @@ metadata:
   namespace: default
 data:
   prod_up: |-
-    apt update; apt -y upgrade; apt -y install --no-install-recommends git g++;
+    apt update; apt -y upgrade; apt -y install --no-install-recommends git g++ pkg-config cmake build-essential;
```
Comment (ypkang) [inline scripts/jaseci.yaml]: Why are these new dependences needed here?

File: scripts/jaseci.yaml [hunk 2]
```diff
@@ -17,11 +17,12 @@ metadata:
...
     git clone https://github.com/Jaseci-Labs/jaseci.git
     cd jaseci;
     cd jaseci_core; source install.sh;
     cd ../jaseci_serv; source install.sh;
+    cd ../jaseci_kit; source install.sh;
```
Comment (ypkang) [inline scripts/jaseci.yaml]: the default jaseci yaml file shouldn't require installing of jaseci_kit. so let's not touch this default yaml file and just make a new one for our experiments need.

**Review by ypkang — CHANGES_REQUESTED** [review]
looks good except for the changes on jaseci.yaml.

File: scripts/jaseci.yaml (hunk 1 above)
Comment (Gorgeous-Patrick) [reply scripts/jaseci.yaml]: Otherwise jaseci kit installer fails.

Comment (ypkang) [reply scripts/jaseci.yaml]: okay i see. Yea let's not commit this to the main repo and just keep this as a separate jaseci yaml file that we use for our experiments.

Comment (Gorgeous-Patrick) [reply scripts/jaseci.yaml]: So i just create another file in script folder and then do the same thing?

Comment (ypkang) [reply scripts/jaseci.yaml]: Yes and lets not commit that to the main jaseci repo. you can just leave it as a unstaged file on your machine.

Comment (Gorgeous-Patrick) [reply scripts/jaseci.yaml]: Ok

**Comment by ypkang** [comment]
@Gorgeous-Patrick can you revert the changes on jaseci.yaml so i can merge this? I would do it myself but I'd assume you want to keep the changes somewhere else.

**Comment by Gorgeous-Patrick** [comment]
Reverted @ypkang

**Review by ypkang — APPROVED** [review]
LGTM

**Pull request merged**